networkx/algorithms/flow/maxflow.py

```diff
@@ -459,8 +459,7 @@ def minimum_cut(flowG, _s, _t, capacity="capacity", flow_func=None, **kwargs):
     partition = (set(flowG) - non_reachable, non_reachable)
     # Finally add again cutset edges to the residual network to make
     # sure that it is reusable.
-    if cutset is not None:
-        R.add_edges_from(cutset)
+    R.add_edges_from(cutset)
```

Comment:
The logic here is that `cutset` is always a list (computed in line 452 above), so `cutset` can never be None, right? This looks right to me but I'm just double-checking that we agree on the reasoning!

Reply:
Yep, that was my reasoning as well!